events/models.py

```diff
@@ -1165,12 +1165,26 @@ def send_published_notification(self, request=None):
         )
 
     def send_draft_posted_notification(self, request=None):
+        if self.is_created_with_apikey:
+            # Events created with an API key shouldn't send draft notifications.
+            return
```

Comment:
Matter of a taste; I think this would benefit a bit of air in a form of line break after returning [None] to make the rest of the logic more "separate"